modin/config/__main__.py

```diff
@@ -11,16 +11,75 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+Content of this file should be executed if module `modin.config` is called.
+
+If module is called (using `python -m modin.config`) configs help will be printed.
+Using `-export_path` option configs description can be exported to the external CSV file
+provided with this flag.
+"""
+
 from . import *  # noqa: F403, F401
 from .pubsub import Parameter
+import pandas
+import argparse
 
 
 def print_config_help():
+    """Print configs help messages."""
     for objname in sorted(globals()):
         obj = globals()[objname]
         if isinstance(obj, type) and issubclass(obj, Parameter) and not obj.is_abstract:
             print(f"{obj.get_help()}\n\tCurrent value: {obj.get()}")  # noqa: T001
 
 
+def export_config_help(filename: str):
+    """
+    Export all configs help messages to the CSV file.
+
+    Parameters
+    ----------
+    filename : str
+        Name of the file to export configs data.
+    """
+    configs = pandas.DataFrame(
+        columns=[
+            "Config Name",
+            "Env. Variable Name",
+            "Default Value",
+            "Description",
+            "Options",
+        ]
+    )
+    for objname in sorted(globals()):
+        obj = globals()[objname]
+        if isinstance(obj, type) and issubclass(obj, Parameter) and not obj.is_abstract:
+            data = {
+                "Config Name": obj.__name__,
+                "Env. Variable Name": obj.varname,
```

Comment:
note that this would stop working as soon as we implement something that is not environment variable-backed...

Reply:
Yes, will be addressed in https://github.com/modin-project/modin/issues/3168.